src/RelayClient.ts

```diff
@@ -53,19 +56,33 @@ type RequestConfig = {
   estimatedGasCorrectionFactor?: BigNumberish;
 };
 
+//FIXME name standardization
 type TokenGasEstimationParams = Pick<
```

Comment:
IMHO having the types defined in this way makes the code very difficult to read.
No need to change it now, but we may probably discuss it with the rest of the team